rules/source-control-tools/rule.md

```diff
@@ -0,0 +1,143 @@
+
+::: greybox
+**🧐 Did You Know?**  
+One of the earliest version control systems, named **SCCS (Source Code Control System)**, was developed by Marc J. Rochkind at Bell Labs in 1972. At that time, the need to efficiently manage changes to source code was just beginning to emerge in the software industry.  
+This system enabled the tracking of revisions for individual files, a significant innovation for its time. It's from these early initiatives that modern version control tools like Git and Mercurial have evolved, offering far more advanced features and collaborative code management capabilities.
+:::
+
+### [GitKraken](https://www.gitkraken.com/)
+
+GitKraken is a renowned suite of developer tools designed to enhance productivity in software development. It has been evolving for nearly a decade, reflecting a continuous commitment to improving the efficiency and effectiveness of developers and their teams. GitKraken began with desktop-centric and IDE-based tools, like the GitKraken Client and GitLens for Visual Studio Code, both of which have become integral to the workflows of tens of millions of developers worldwide​​.
+
+The company behind GitKraken, Axosoft, has been recognized for its significant contributions to the developer community. For instance, Eric Amodio, the Chief Technology Officer at GitKraken, was awarded the Microsoft MVP Award for Developer Technologies in 2023. This award highlights his exceptional contributions to the developer community, particularly through GitLens, a popular Git extension for Visual Studio Code created by Amodio in 2016. GitLens has become a crucial tool in the daily coding processes of millions of developers, with its installation numbers surpassing 23 million.
+
+#### Pros:
+
+-   **Intuitive Interface:** GitKraken's graphical user interface is visually appealing and easy to understand, simplifying interactions with Git&#8203;
+-   **Integration with Platforms:** Smooth integration with GitHub, GitLab, and Bitbucket, enhancing workflow efficiency
+-   **Merge Conflict Editor:** Built-in tool for resolving merge conflicts, showcasing conflicting segments side-by-side
+-   **Drag-and-Drop Functionality:** Enables performing many Git operations easily with just drag-and-drop
+-   **Profile Management:** Useful for managing multiple profiles, distinguishing between personal and professional work
+-   **Dark and Light Themes:** Catering to various visual preferences
+-   **Extensive Documentation and Tutorials:** Assists new users in quickly becoming proficient with the tool
+-   **Cross-Platform Compatibility:** Available on Windows, MacOS, and Linux
+-   **Graphical Commit History:** Offers a vivid representation of repository history, enhancing the understanding of project flow
+-   **Powerful Search Functionality:** Makes finding commits, branches, and other elements straightforward
+
+#### Cons:
```

Comment:
@PothieuG 

- [ ]  1. Please add a red cross here ❌

Same for the other Cons